backend/app/services/base.py

```
@@ -0,0 +1,49 @@
+"""Base service for business logic operations."""
+
+from typing import Any, Dict, Generic, List, Optional, TypeVar
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)